docs/lib/webGLcurve.js

```diff
@@ -366,83 +297,84 @@ function draw_connected_full_view_proportional(num) {
 function draw_3D_connected(num) {
   return function(func) {
     return generateCurve('none', 'lines', num, func, '3D')
+    //requestAnimationFrame(generateCurve)
   }
 }
 
 /**
  * returns a function that turns a given 3D Curve into a Drawing, 
- * by sampling the 3D Curve at <CODE>num</CODE> sample points. 
- * The Drawing consists of isolated points, and does not connect them. 
+ * by sampling the 3D Curve at <CODE>num</CODE> sample points 
+ * and connecting each pair with a line. 
  * When a program evaluates to a Drawing, the Source system 
  * displays it graphically, in a window, instead of textually. 
- * The parts between (0,0,0) and (1,1,1) of the resulting 
- * Drawing are shown within the cube representing the 3D space. 
+ * The Drawing is stretched or shrunk 
+ * to show the full curve 
+ * and maximize its width and height within the cube.
```

Comment:
...and depth?

Reply:
I have updated teh jsdoc and pushed the commit.